docs/modules/operation/pages/provisioning/hardware-inventory.adoc

```diff
@@ -0,0 +1,113 @@
+[[hardware-inventory-plugin]]
+= Hardware Inventory Plugin
+
+== Introduction
+
+{page-component-title} can be extended to import Entity-MIB information from SNMP agents to enrich node data.
+There are two types of Entity-MIBs.
+The first type is a generic implementation of https://tools.ietf.org/html/rfc4133[RFC 4133 ENTITY-MIB].
+The second type are vendor-specific Entity-MIBs, which some manufacturers provide.
+
+These Entity-MIBs provide dynamic attributes to describe device components.
+This includes information such as model number, description, serial number, and more.
+The optional hardware inventory extension of the Provisiond service will add these fields to nodes in {page-component-title}.
+
+== Installation and configuration
+
+IMPORTANT: To use this feature you have to install the SNMP hardware inventory plugin from the repository.
+
+****
+To install the opennms-plugin-provisioning-snmp-hardware-inventory package:
+
+* For RPM-based distributions:
+** `yum install opennms-plugin-provisioning-snmp-hardware-inventory`
+* For Debian-based distributions:
+** `apt-get install opennms-plugin-provisioning-snmp-hardware-inventory`
+****
+
+Once enabled, the hardware inventory plugin will collection the ENTITY-MIB::entPhysicalTable of all devices without any configuration.
+If you want to collect inventory from devices that provide additional fields, you can specify which OIDs to use in the configuration file `$OPENNMS_HOME/etc/snmp-hardware-inventory-adapter-configuration.xml`.
+This file includes support for the https://tools.cisco.com/Support/SNMP/do/BrowseMIB.do?mibName=ENTITY-MIB[CISCO-ENTITY-MIB] as a starter example.
+
+Any MIB column indexed by `entPhysicalIndex` can be added to the configuration file and the hardware inventory adapter will collect the value and store it for each entity that contains a non-null value for the custom attribute.
+The `hw-extension` element lets you group a list of Entity-MIB extensions defined in `mibObj` elements based on a sysOidMask of a node.
+If the system enterprise ID of the node matches the `sysOidMask` defined on the `hw-extension`, the `mibObj` entities inside are collected for populating the inventory.
+When a vendor provides the standard objects on a different table rather than `entPhysicalTable`, you can override one of the base attributes by specifying on the `mibObj` that the attribute that must be overridden.
```

Comment:
```suggestion
When a vendor provides the standard objects on a different table rather than `entPhysicalTable`, you can override one of the base attributes by specifying on the `mibObj` that the attribute must be overridden.
```

Reply:
```suggestion
When a vendor provides the standard objects on a table other than `entPhysicalTable`, you can override any of the base attributes by specifying which attribute is to be overridden.
```